src/author-tags.js

```diff
@@ -7,13 +7,28 @@ import { get } from "lodash";
  * @param {*} seoConfig
  * @param {...*} params See {@link Generator} for other Parameters
  */
+
 export function AuthorTags(seoConfig, config, pageType, data, { url }) {
-  if (pageType != "story-page" || pageType != "story-page-amp") return [];
+  if (pageType === "story-page" || pageType === "story-page-amp") {
+    return [
+      {
+        name: "twitter:creator",
+        content: getTwitterCreator(data),
+      },
+    ];
+  }
+  return [];
+}
 
-  return [
-    {
-      name: "twitter:creator",
-      content: get(data, ["data", "story", "author-name"]),
-    },
-  ];
+function getTwitterCreator(storyData) {
+  const twitterData = get(storyData, ["data", "story", "authors", 0, "social", "twitter"], {});
+  const twitterUrl = twitterData.url || "";
+  const twitterHandle = twitterData.handle || "";
+  if (twitterHandle.startsWith("@")) return twitterHandle;
+  const twitterHandleFromUrl = twitterUrl.split("/").pop();
+  if (twitterHandleFromUrl) {
+    if (twitterHandleFromUrl.startsWith("@")) return twitterHandleFromUrl;
+    return `@${twitterHandleFromUrl}`;
+  }
```

Comment:
can we add this as separate util function as its used in another place as well

Reply:
There isn't much code reuse. Also it will be more legible to have it this way